utils/nctl/sh/assets/setup.sh

```diff
@@ -86,16 +84,8 @@ function _set_nodes()
             "cfg['rest_server']['address']='0.0.0.0:$(get_node_port_rest "$IDX")';"
             "cfg['rpc_server']['address']='0.0.0.0:$(get_node_port_rpc "$IDX")';"
             "cfg['event_stream_server']['address']='0.0.0.0:$(get_node_port_sse "$IDX")';"
-            "toml.dump(cfg, open('$PATH_TO_FILE', 'w'));"
-        )
-
-        if [ ! -z "$SPECULATIVE_EXEC_ADDR" ]; then
-            SCRIPT+=(
-                "cfg['rpc_server']['speculative_execution_address']='0.0.0.0:$(get_node_port_speculative_exec "$IDX")';"
-            )
-        fi
-
-        SCRIPT+=(
+            "cfg['speculative_exec_server']['address']='0.0.0.0:$(get_node_port_speculative_exec "$IDX")';"
```

Comment:
I know it's not a scope of this PR, but can you consider extending the `nctl-view-node-ports` to also show the speculative exec port number?

Reply:
Done in [e75c9c8](https://github.com/casper-network/casper-node/pull/3972/commits/e75c9c8098f48970f4cd0722d670a3799f60f995).